libraries/dotnet/EnergyOriginEventStore/EnergyOriginEventStore/EventStore/Memory/MessageEventArgs.cs

```diff
@@ -0,0 +1,16 @@
+
+namespace EnergyOriginEventStore.EventStore.Memory;
+
+internal class MessageEventArgs : EventArgs
+{
+    public readonly string Message;
+    public readonly string Topic;
+    public readonly MemoryPointer Pointer;
+
+    internal MessageEventArgs(string message, string topic, MemoryPointer pointer)
+    {
+        Message = message;
+        Topic = topic;
+        Pointer = pointer;
```

Comment:
```suggestion
        Message = message ?? throw new ArgumentNullException(nameof(message));
        Topic = topic ?? throw new ArgumentNullException(nameof(topic));
        Pointer = pointer ?? throw new ArgumentNullException(nameof(pointer));
```

Reply:
This could avoided by treating this as an error and not a warning. Add the following to editorconfig

```
# CS8625: Cannot convert null literal to non-nullable reference type.
dotnet_diagnostic.CS8625.severity = error
```

Reply:
Issue created